### PR TITLE
[5.2] [Guided tours] Add missing auto start parameter in the user interface of tours

### PR DESCRIPTION
**Pull request by obuisard**
### Summary of Changes

The autostart parameter has been introduced in Joomla 5.1 to facilitate the autostart feature and mainly the welcome tour.

This PR goes a step further and adds the ability to make a tour auto-start-able in the interface.
The purpose of the PR is to just add the ability to modify the parameter from the edit view.
The functionality for auto-starting tours is presented in a separate PR #43814.

image

### Testing Instructions

Go to System -> Guided Tours.
Pick a tour.
Check that the option is present.
Set it on.
Duplicate the tour.
Make sure the duplicated tour replicates the original tour auto-start value.

Note. The welcome tour, at present (until the addition of the joint PR), if set to auto-start, will run once and be automatically set to auto-start: false after the run.
Tours set to auto start will not start automatically with this PR alone.

### Actual result BEFORE applying this Pull Request

No auto start parameter.

### Expected result AFTER applying this Pull Request

The auto start parameter can be changed from the tour edit view.

### Link to documentations
Please select:
- [x ] Documentation link for docs.joomla.org: [Edit Tour](https://docs.joomla.org/Help5.x:Guided_Tours:_New_or_Edit_Tour)
- [ ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: administrator/components/com_guidedtours/forms/tour.xml [hunk 1]
```diff
@@ -160,4 +160,16 @@
...
 		<option value="*">JALL</option>
 	</field>
+
+	<field
+		name="autostart"
+		type="radio"
+		layout="joomla.form.field.radio.switcher"
+		label="COM_GUIDEDTOURS_AUTOSTART_LABEL"
+		description="COM_GUIDEDTOURS_AUTOSTART_DESC"
+		default="0"
```
Comment (brianteeman) [inline administrator/components/com_guidedtours/forms/tour.xml]:
```suggestion
		name="autostart"
		type="radio"
		label="COM_GUIDEDTOURS_AUTOSTART_LABEL"
		description="COM_GUIDEDTOURS_AUTOSTART_DESC"
		layout="joomla.form.field.radio.switcher"
		default="0"
```

codestyle

Comment (obuisard) [reply administrator/components/com_guidedtours/forms/tour.xml]: Thanks Brian.

**Comment by RickR2H** [comment]
I have tested this item :white_check_mark: successfully on 535cd9aa1168ccbe8463d63436d346544b836ccb<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43690">issues.joomla.org/tracker/joomla-cms/43690</a>.</sub>

**Comment by hans2103** [comment]
I have tested this item :white_check_mark: successfully on 535cd9aa1168ccbe8463d63436d346544b836ccb<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43690">issues.joomla.org/tracker/joomla-cms/43690</a>.</sub>

**Comment by Quy** [comment]
@obuisard Change Draft status?

**Comment by obuisard** [comment]
> @obuisard Change Draft status?

We are waiting for the second PR that complements it. I don't want this one to be merged before that Thank you @Quy

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43690">issues.joomla.org/tracker/joomla-cms/43690</a>.</sub>

**Comment by Hackwar** [comment]
Thank you for your contribution! 😃

**Pull request merged**